index.js

```diff
@@ -0,0 +1,142 @@
+            handlerInput.requestEnvelope.request.intent.name === 'AMAZON.YesIntent'
+    },
+
+    async handle(handlerInput) {
+        const activity = await getSuggestion()
+        return handlerInput.responseBuilder
+            .speak(activity)
+            .reprompt('would you like a suggestion?')
+            .withShouldEndSession(false)
+            .getResponse()
+    },
+}
+const ActivityIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+            && handlerInput.requestEnvelope.request.intent.name === 'ActivityIntent'
+    },
+    async handle(handlerInput) {
+        const activity = await getSuggestion()
+        const reprompt = 'Would you like another suggestion?'
+
+        return handlerInput.responseBuilder
+            .speak(activity + reprompt)
+            .reprompt(reprompt)
+            .withShouldEndSession(false)
+            .getResponse()
+    },
+}
+
+const CancelAndStopIntenthandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+            && (handlerInput.requestEnvelope.request.intent.name === 'AMAZON.CancelIntent'
+                || handlerInput.requestEnvelope.request.intent.name === 'AMAZON.StopIntent')
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .withShouldEndSession(true)
+            .getResponse()
+    },
+}
+const HelpIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+            && handlerInput.requestEnvelope.request.intent.name === 'AMAZON.HelpIntent'
+    },
+    handle(handlerInput) {
+        const guidance = 'We can provide suggestions for things to do!'
+        const reprompt = 'Would you like another suggestion?'
+
+        return handlerInput.responseBuilder
+            .speak(guidance + reprompt)
+            .reprompt(reprompt)
+            .withShouldEndSession(false)
+            .getResponse()
+    },
+}
+const FallBackIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+            && handlerInput.requestEnvelope.request.intent.name === 'AMAZON.FallBackIntent'
+    },
+    handle(handlerInput) {
+        const guidance = 'say what?'
+        const reprompt = 'Would you like another suggestion?'
+
+        return handlerInput.responseBuilder
+            .speak(guidance + reprompt)
+            .reprompt(reprompt)
+            .withShouldEndSession(false)
+            .getResponse()
+    },
+}
+const NavigateHomeIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+            && handlerInput.requestEnvelope.request.intent.name === 'AMAZON.NavigateHomeIntentHandler'
+    },
+    handle(handlerInput) {
+        const reprompt = 'Have fun!'
```

Comment:
You usually want NavigateHome to be similar to Launch since that's the "home" of your skill.